components/AppBar/tests/unit/MDCAppBarViewControllerTests.m

```diff
@@ -89,4 +89,22 @@ - (void)testHeaderViewElevationSetByShadowIntensityChange {
   XCTAssertTrue(blockCalled);
 }
 
+- (void)testShouldAdjustHeightBasedOnHeaderStackViewSetToYES {
+  // Given
+  MDCAppBarViewController *appBarController = [[MDCAppBarViewController alloc] init];
+  CGFloat navigationBarHeight =
+      [appBarController.navigationBar sizeThatFits:appBarController.headerView.frame.size].height;
+  UIView *bottomBar = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 100, 200)];
+  appBarController.headerStackView.bottomBar = bottomBar;
+
+  // When
+  appBarController.shouldAdjustHeightBasedOnHeaderStackView = YES;
+
+  // Then
+  CGFloat minHeight = appBarController.headerView.minimumHeight;
+  CGFloat maxHeight = appBarController.headerView.maximumHeight;
+  XCTAssertEqual(minHeight, maxHeight);
+  XCTAssertEqualWithAccuracy(minHeight, 200 + navigationBarHeight, 0.01);
```

Comment:
Pull 200 out to a constant in the `Given` section to clarify that it's semantically the same value here and above.

Reply:
Addressed in 1b97d553ff0a58d5597bbe7382a338085125677f.